packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -63,14 +63,16 @@ class PostsRepo extends AbstractRepo<"Posts"> {
   }
 
   async postRouteWillDefinitelyReturn200(id: string): Promise<boolean> {
+    const maybeRequireAF = isAF ? "AND af=true" : ""
```

Comment:
Might be a pre-existing issue, but should this be `af IS TRUE`?

Reply:
AFAIK these are equivalent (I tried `select count(*) from "Posts" where af=true` and the same with `is true` just now, and got the same result). We do seem to be favoring IS TRUE as a convention, though, so I'll change it for consistency's sake.

Reply:
The difference is that `SELECT NULL IS TRUE` returns `FALSE`, but `SELECT NULL = TRUE` returns `NULL`.